src/sass/base/_va.scss

```diff
@@ -215,11 +215,11 @@ table {
   clear: both;
   margin: 0;
   padding: 0;
-  min-height: 147.19px;
+  min-height: 167.19px;
```

Comment:
Could we add a comment on this change and the next one saying that they're used to set the header height so that it doesn't flash when the menus are rendered? I should have done that when I initially made that change.

Reply:
Good idea, will do!